packages/esm-patient-common-lib/src/dashboards/createDashboardLink.tsx

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
 import { DashboardLinkConfig } from '../types';
 import { DashboardExtension } from './DashboardExtension';
 
 export const createDashboardLink = (db: DashboardLinkConfig) => {
   const Dashboard = ({ basePath }: { basePath: string }) => {
-    return <DashboardExtension title={db.title} basePath={basePath} />;
+    return (
+      <BrowserRouter>
```

Comment:
Instead of creating a browser router wrapper for each dashboard link, wouldn't it be better to wrap the entire left nav once?

Reply:
Maybe, but to do that we'd have to pass the route information as extension state, alongside `basePath`. I'm neither for nor against doing that, but that's what would need to happen.

Reply:
@brandones @ZacButko I moved the BrowserRouter to the entire left nav and passed the route props to the extension state it is working well. We need to also merge [this PR](https://github.com/openmrs/openmrs-esm-core/pull/457) to get the work to show well. I need your approval on the other PR